Core/src/org/sleuthkit/autopsy/core/UserPreferences.java

```diff
@@ -369,4 +372,26 @@ public static void setAppName(String name) {
         preferences.put(APP_NAME, name);
     }
 
+    /**
+     * get the maximum number of log files to save
+     * @return Number of log files
+     */
+    public static int getLogFileCount() {
+        return preferences.getInt(MAX_NUM_OF_LOG_FILE, LOG_FILE_NUM_INT);
+    }
+    
+    /**
+     * get the default number of log files to save
+     * @return LOG_FILE_COUNT
+     */
+    public static String getDefaultLogFileCount() {
+        return LOG_FILE_NUM_STRING;
+    }
```

Comment:
Let's return an int here and let clients change it to a string if they want to; it is more natural to return an int.